backend/src/routes.js

```diff
@@ -90,12 +95,12 @@ router.post('/ngos', jwtMiddleware({ secret: process.env.ACCESS_TOKEN_KEY, algor
         image,
         webpage,
         description,
-        main_representative,
```

Comment:
These are coming from the frontend so we need to keep the original names with the underscores. Feel free to remove the eslint rule that caused this. Sorry about that :(